### PR TITLE
Add Navitia development docker

### PR DESCRIPTION
**Pull request by GuillaumeLescure**

### CODE REVIEW

File: nav_dev/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    && wget -P /tmp/source/tyr https://raw.githubusercontent.com/CanalTP/navitia/dev/source/tyr/requirements_dev.txt \
+    && wget -P /tmp/source/monitor https://raw.githubusercontent.com/CanalTP/navitia/dev/source/monitor/requirements.txt \
+    && wget -P /tmp/source/jormungandr https://raw.githubusercontent.com/CanalTP/navitia/dev/source/jormungandr/requirements.txt \
+    && wget -P /tmp/source/jormungandr https://raw.githubusercontent.com/CanalTP/navitia/dev/source/jormungandr/requirements_dev.txt \
+    && wget -P /tmp/source/sql https://raw.githubusercontent.com/CanalTP/navitia/dev/source/sql/requirements.txt
+
+# Create Python virtual environment
+RUN cd /home/nav-dev \
+    && su - nav-dev -c "/bin/bash -c \"source /usr/share/virtualenvwrapper/virtualenvwrapper.sh && mkvirtualenv -p /usr/bin/python2 nav-py2 && pip install -r /tmp/source/eitri/requirements.txt && pip install -r /tmp/source/cities/requirements.txt && pip install -r /tmp/source/tyr/requirements_dev.txt && pip install -r /tmp/source/monitor/requirements.txt && pip install -r /tmp/source/jormungandr/requirements_dev.txt && pip install -r /tmp/source/sql/requirements.txt\"" \
```
Comment (pbougue) [inline nav_dev/Dockerfile]: So you dev on a different venv than the one used on platforms, this does look easier, but that's probably not a good thing.
What if you use some new lib in tyr and it was already in jormun requirements?
- Your tests won't crash so you won't add the lib to tyr requirements, and you will only realize it on CI (best case) or on deploy/run.

Comment (GuillaumeLescure) [reply nav_dev/Dockerfile]: I'm not sure to understand that. The "nav-py2" created by the Dockerfile is used by the user in the docker. It's the only venv you need to use.

Indeed you have to update it, if the "requirements.txt" are modified. The same way you manually have to do it. But in that case, the container can be updated and so a "docker pull" can fix the issue.

Comment (GuillaumeLescure) [reply nav_dev/Dockerfile]: Moreoever, the function "navitia_env_update" can do it for you

Comment (GuillaumeLescure) [reply nav_dev/Dockerfile]: At the end, the risk is acceptable for a development environment. You don't have to use it and you can make a virtual environment for each project if you prefere to match the production.

**Pull request closed without merging**